vignettes/vignette_best_practices.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Best practices when writing vignette in admiral"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Best practices when writing vignette in admiral}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup0, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This guidance ensures consistency across the vignettes in the admiral package in terms of content, structure and code execution. 
+As a general rule, the vignette workflow defined in [r-pkgs.org](https://r-pkgs.org/vignettes.html) is followed.
+
+
+# Metadata
+
+Each vignette in admiral should start with the following metadata.
+
+```
+---
+title: "<Title>"
+output:  rmarkdown::html_vignette:
+vignette: >
+  %\VignetteIndexEntry{<Title>}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+```
+
+The `<Title>` of the vignette should be meaningful. 
+
+# Markdown
```

Comment:
I would add some text after the headings. Headings without text look a bit strange.